packages/core/src/components/accordion/components/accordion-section-theme-neutral.scss

```diff
@@ -34,14 +34,33 @@
     }
   }
 
-  .dso-section-body {
+  .dso-section-body:not(.dso-hide) {
+    background-color: transparent;
+    border: 1px solid accordion.$neutral-border-color;
+    box-shadow: 1px 3px 4px rgba(0, 0, 0, 0.5);
+    margin-block: (accordion.$border-radius * -1) 0;
+
+    &.dso-animate-ready {
+      transition-property: grid-template-rows, margin-block, border;
+    }
+
+    .dso-section-body-content {
+      padding: units.$u2 units.$u2 units.$u3;
+
+      @media screen and (min-width: media-query-breakpoints.$screen-xs-min) {
+        padding: units.$u2 units.$u4 units.$u3;
+      }
+    }
+  }
+
+  .dso-section-body[open] {
```

Comment:
^^ deze twee selectors kunnen m.i. gecombineerd worden, en let op de overlap veroorzaakt door `:not()`.